content/4_Gold/PURS.mdx

```diff
@@ -160,241 +160,307 @@ public class segtree {
 </Resources>
 
 <LanguageSection>
-
 <CPPSection>
 
-Note that `st.init(n+1)` allows us to update and query indices in the range
-`[0,n+1)=[0,n]`.
-
 ```cpp
-#include <bits/stdc++.h>
-using namespace std;
+#include <iostream>
+#include <cassert>
+#include <vector>
+#include <algorithm>
+
+using std::cout;
+using std::endl;
+using std::vector;
+
+/** A data structure that can answer point update & range minimum queries. */
```

Comment:
```suggestion
// A data structure that can answer point update & range minimum queries.
```

Reply:
https://stackoverflow.com/questions/20869421/is-there-a-way-to-write-a-single-line-javadoc-comment-like-in-doxygen